packages/wrangler/turbo.json

```diff
@@ -21,55 +21,61 @@
 				"config-schema.json"
 			],
 			"env": [
-				"NODE_EXTRA_CA_CERTS",
-				"CLOUDFLARE_ACCOUNT_ID",
-				"SOURCEMAPS",
-				"NODE_ENV",
-				"SPARROW_SOURCE_KEY",
 				"ALGOLIA_APP_ID",
 				"ALGOLIA_PUBLIC_KEY",
-				"CLOUDFLARE_API_TOKEN",
-				"CLOUDFLARE_ACCOUNT_ID",
-				"WRANGLER_AUTH_DOMAIN",
-				"PATH",
-				"WRANGLER_LOG",
-				"EXPERIMENTAL_MIDDLEWARE",
-				"FORMAT_WRANGLER_ERRORS",
+				"NODE_ENV",
+				"SOURCEMAPS",
+				"SPARROW_SOURCE_KEY",
+				"SENTRY_DSN"
+			],
+			"passThroughEnv": [
```

Comment:
A list of environment variables that are made available to tasks. Any change to these environment variables will not affect how tasks miss/hit the cache

Reply:
The env vars here are only actually used at runtime (i.e. when you run `wrangler` the CLI tool). So they do not affect the build output at all, and therefore there is no need to include these in the cache key.

Notably we previously had `PWD` in the cache key, which meant that just changing directory and running the wrangler build again would miss the cache.